zabapgit.prog.abap

```diff
@@ -1205,6 +1207,7 @@ CLASS lcl_xml IMPLEMENTATION.
 *    buffer serializer metadata. Git node will be removed lateron
     ms_metadata-class = li_element->get_attribute_ns( c_attr_serializer ).
     ms_metadata-version = li_element->get_attribute_ns( c_attr_serializer_version ).
+    ms_metadata-late_deser = li_element->get_attribute_ns( c_attr_late_deserialize ).
```

Comment:
I dont think we need the flag in the serialized XML, we can just let the code always decide if it should do late deserialization.

Call the create_object method, on this method the get_metadata can be called to get the late deserialization flag, without saving it in the XML